internal/manager/manager.go

```diff
@@ -68,6 +69,8 @@ var (
 	printDefaults                 = flag.Bool("printDefaults", false, "print all variables with their default values and exit")
 	printFormat                   = flag.String("printFormat", "table", "output format for --printDefaults. Can be table, json, yaml or list")
 	promCRDResyncPeriod           = flag.Duration("controller.prometheusCRD.resyncPeriod", 0, "Configures resync period for prometheus CRD converter. Disabled by default")
+	clientQPS                     = flag.Int("client.qps", 5, "defines K8s client QPS")
+	clientBurst                   = flag.Int("client.burst", 10, "defines K8s client burst")
```

Comment:
Please mention this changes at changelog and update linked issue with possible workaround solution. 

these are not obvious flags to use. It'd be great to have an article link when it should be modified.